Packs/Tanium/Integrations/Tanium/Tanium.py

```diff
@@ -7,11 +6,9 @@
 
 from cStringIO import StringIO
 
-if not demisto.params()['proxy']:
-    del os.environ['HTTP_PROXY']
-    del os.environ['HTTPS_PROXY']
-    del os.environ['http_proxy']
-    del os.environ['https_proxy']
+handle_proxy(demisto.params().get('proxy'))
+
+response = ''
```

Comment:
?

Reply:
"response" is defined globally in this integration, I added a `main` function here for unit tests so I had to move the definition of the variable outside.